shared/templates/yamlfile_value/oval.template

```diff
@@ -45,13 +45,11 @@
     <ind:value datatype="string" operation="equals" var_ref="{{{ XCCDF_VARIABLE }}}"/>
   </ind:variable_state>
 
-  {{% for val in VALUES %}}
-  <local_variable id="local_variable_{{{ rule_id }}}" datatype="string" comment="comment1" version="1">
-    <regex_capture pattern='{{{ val.value }}}'>
-       <object_component item_field="value" record_field="#" object_ref="object_{{{ rule_id }}}" />
+  <local_variable id="local_variable_{{{ rule_id }}}" datatype="string" comment="Captured value to be compared with XCCDF value" version="1">
+    <regex_capture pattern='{{{ (VALUES|first).value }}}'>
+       <object_component item_field="value" record_field="{{{ (VALUES|first).key|default('#') }}}" object_ref="object_{{{ rule_id }}}" />
```

Comment:
uhm... wait, so this removes support for using multiple values?

Reply:
Only when the template data is using XCCDF variables. The previous use case of multiple values is preserved.

Reply:
Is there a use case where you'd like to compare an XCCDF Variable against multiple values?
Even in the non-embedded case?

I think it would be possible to do so, but id would add more complexity to this template :grimacing:

Reply:
I don't see a use-case now. Let's expand once it's needed.